src/ace1_compat.jl

```diff
@@ -1,390 +1,276 @@
-# This module is a translation of most ACE1x options to the new ACE 
-# kernels. It is used to provide compatibility with the old ACE1 / ACE1x. 
-
 module ACE1compat
 
 using NamedTupleTools, StaticArrays, Unitful
-import ACEpotentials: DefaultHypers, Models 
```

Comment:
as a general rule, one should try to not make whitespace changes to repositories you don't own. By adding or removing whitespace you make it much harder to review your PR.